components/dashboard/src/settings/Preferences.tsx

```diff
@@ -71,63 +187,58 @@ export default function Preferences() {
         setTheme(theme);
     }
 
+    const browserIdeOptions = orderedIdeOptions(ideOptions, "browser");
+    const desktopIdeOptions = orderedIdeOptions(ideOptions, "desktop");
+
     return <div>
         <PageWithSubMenu subMenu={settingsMenu} title='Preferences' subtitle='Configure user preferences.'>
             <h3>Default IDE</h3>
-            <p className="text-base text-gray-500">Choose which IDE you want to use.</p>
-            <div className="mt-4 space-x-4 flex">
-                <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultIde === 'code'} onClick={() => actuallySetDefaultIde('code')}>
-                    <div className="flex justify-center mt-3">
-                        <img className="w-16 filter-grayscale self-center" src={vscode} />
-                    </div>
-                </SelectableCard>
-                <Tooltip content={'Early access version, still subject to testing.'} >
-                    <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultIde === 'code-latest'} onClick={() => actuallySetDefaultIde('code-latest')}>
-                        <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={vscodeInsiders} />
-                        </div>
-                        <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Insiders</PillLabel>
-                    </SelectableCard>
-                </Tooltip>
-            </div>
-            <div className="mt-4 space-x-4 flex">
-                <CheckBox
-                    title={<div>Open in Desktop IDE <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></div>}
-                    desc="Choose whether you would like to open your workspace in a desktop IDE instead."
-                    checked={useDesktopIde}
-                    onChange={(evt) => actuallySetUseDesktopIde(evt.target.checked)} />
+            <p className="text-base text-gray-500 dark:text-gray-400">Choose which IDE you want to use.</p>
+            <div className="my-4 space-x-4 flex">
+                {
+                    browserIdeOptions.map(([id, option]) => {
+                        const selected = defaultIde === id;
+                        const onSelect = () => actuallySetDefaultIde(id);
+                        return renderIdeOption(option, selected, onSelect);
+                    })
+                }
             </div>
-            {useDesktopIde && <>
+            {ideOptions.options[defaultIde].notes &&
+                <InfoBox className="my-5 max-w-2xl"><ul>
+                    {ideOptions.options[defaultIde].notes?.map((x, idx) => <li className={idx > 0 ? "mt-2" : ""}>{x}</li>)}
+                </ul></InfoBox>
+            }
```

Comment:
**praise**: Nice! Just noticed the hiding alert component for VS Code options. Thanks @corneliusludmann! 🎩